content/id/examples/application/deployment-scale.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+kind: Deployment
+metadata:
+  name: nginx-deployment
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: 4 # Update the replicas from 2 to 4
```

We can translate the description also.
```suggestion
  replicas: 4 # Memperbarui replica dari 2 menjadi 4
```